<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactInfoTitleWidget</class>
 <widget class="QWidget" name="ContactInfoTitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>图</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_addressTitle">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>52</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_address">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>52</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1111111</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_copy">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_transfer">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>22</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
